.github/workflows/build.yaml

```diff
@@ -35,3 +35,9 @@ jobs:
       - name: test run
         run: mkdir rpki-cache rpki; src/rpki-client -v -t afrinic.tal -t apnic.tal -t lacnic.tal -t ripe.tal -d rpki-cache rpki
         if: matrix.compiler == 'cc'
+      - name: test run rrdp single ta
```

Comment:
Why do a single ta and then all other ta next?

Reply:
there might be 2 bugs, so 2 separate invocations might highlight what is what